Liquid_Biopsy_Duplex_Analysis/GenerateDuplexConsensusBams/GenerateDuplexConsensusBams.wdl

```diff
@@ -1,3 +1,5 @@
+# Import BaitSetNameCheck task
+import "https://raw.githubusercontent.com/broadinstitute/TAG-public/master/checkBaitSetName/checkBaitSetName.dev.wdl" as checkBaitSetName
```

Comment:
I propose using: https://raw.githubusercontent.com/broadinstitute/TAG-public/v1.1.1/checkBaitSetName/checkBaitSetName.dev.wdl

Reply:
Instead, can you use something like, import "../../../../../tasks/broad/JointGenotypingTasks.wdl" as Tasks

This way we are referring to the current version (but not the master).